deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,90 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+/* #C16E70 old rose #F2F3D9 beige #030027 blue*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    width: 100vw;
+    height: 100vh;
+    flex-direction: column;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background: #F2F3D9;
+    text-align: center;
+    flex-shrink: 1;
+    flex-grow: 0;
+}
+
+header {
+    display: flex;
+    flex-basis: 5vw;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #030027;
+    color: white;
+    flex-shrink: 0;
+    flex-grow: 0;
+
+}
+
+.header-h3{
+    margin: 40px;
+}
+
+ul{
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    gap: 50px;
+    margin: 40px;
+}
+
+div{
```

Comment:
Ik denk dat het beter is om de div elementen een class te geven en de class als selector te gebruiken in je css. Voor nu maakt het niet uit, maar als je grotere websites gaat bouwen wil je niet dat elk div element aangepast wordt.